packages/admin-ui-components/webpack.config.js

```diff
@@ -98,6 +98,7 @@ module.exports = {
         test: /\.js$/,
         loader: "source-map-loader",
       },
+      { test: /\.css$/, use: [ "style-loader", "css-loader" ] },
```

Comment:
Where are we loading raw CSS?

Reply:
Components which depend on Antd styles, have to load its CSS styles explicitly and we have Table component which depends on it:
https://github.com/cockroachdb/ui/pull/162/files#diff-70a6adf902647bd71cee4e2b65f55a1a999736c39698aef2ba64f1a70aedbadaR6